.ci/go-static-checks.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+set -e
+
+PATH=${PATH}:${GOPATH}/bin
+
+# Perform static go tests.
+
+function usage {
+	echo "Usage $0 [OPTIONS] [PACKAGES]"
+	echo "Perform static go checks on PACKAGES (./... by default)."
+	echo
+	echo "List of options:"
+	echo "  -h, --help             print this help"
+	echo "  -n, --no-network       do not access the network"
+}
+
+for i in "$@"; do
+	case $i in
+		-h|--help)
+			usage
+			exit 0
+			;;
+		-n|--no-network)
+			NONETWORK=1
+			shift
+			;;
+		*)
+			args="$args $i"
+			;;
+	esac
+done
+
+go_packages=$args
+
+[ -z "$go_packages" ] && {
+	go_packages=$(go list ./... | grep -v vendor |\
+		    sed -e 's#.*/cc-oci-runtime/#./#')
```

Comment:
That was to work around oddities in how we setup the 2.x repo for golang in travis (because 2.x was a C/C++ project and Travis didn't have a GOPATH setup for us). We may well be able to remove that `sed`.